pkg/scheduler/framework/plugins/interpodaffinity/filtering.go

```diff
@@ -246,7 +246,7 @@ func (pl *InterPodAffinity) PreFilter(ctx context.Context, cycleState *framework
 	if allNodes, err = pl.sharedLister.NodeInfos().List(); err != nil {
 		return framework.NewStatus(framework.Error, fmt.Sprintf("failed to list NodeInfos: %v", err))
 	}
-	if havePodsWithAffinityNodes, err = pl.sharedLister.NodeInfos().HavePodsWithAffinityList(); err != nil {
+	if havePodsWithAffinityNodes, err = pl.sharedLister.NodeInfos().HavePodsWithRequiredAntiAffinityList(); err != nil {
```

Comment:
The line 177 can be change as 

`
for _, existingPod := range nodeInfo.PodsWithRequiredAntiAffinity{
			topoMap.updateWithAntiAffinityTerms(pod, node, existingPod.RequiredAntiAffinityTerms, 1)
}
`

Reply:
thanks for the catch